### DISCUSSION TITLE
Any tips on running in airplane mode?

### DISCUSSION THREAD

**Discussion by cjbland** [remix-run/remix#2140] [Q&A] [answered]
I was hoping to get some dev going on one of my flights, but running `npm run dev` gives me this error with my laptop in airplane mode:

> [1] Watching Remix app in development mode...
[1] Could not find an IPv4 address.

Are there any tips to get this to work or do I really need a valid IP address? For what it's worth, my `create-react-app` projects were able to start and serve on localhost. Thanks!

**Comment by sergiodxa** [comment, marked as answer]
Is that error making the process stop? Because if it’s not it should be running on localhost too

**Reply by cjbland** [reply]
Yes, it says it is exiting. I don’t have my laptop in front of me, but if you turn WiFi off you should see it. I also updated to 1.2.2 of remix as well.

**Reply by sergiodxa** [reply]
I tried to replicate it and I couldn't, I disabled my WiFi and I got the address using localhost.

**Reply by cjbland** [reply]
Thanks for your help with this... I'm so sorry...I did not realize that `@remix-run/dev` wasn't updated to 1.2.2 when I updated the other (non-dev) dependencies. Once I properly updated that it now starts the server successfully without an active network connection. Sorry again!
